api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -148,4 +149,7 @@ message TcpProxy {
   // The maximum number of unsuccessful connection attempts that will be made before
   // giving up. If the parameter is not specified, 1 connection attempt will be made.
   google.protobuf.UInt32Value max_connect_attempts = 7 [(validate.rules).uint32 = {gte: 1}];
+
+  // Optional configuration for TCP proxy hash policy.
```

Comment:
Can you describe what the behavior is if this is not set? Also, can you briefly describe that this is currently limited to a single hash policy?